portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,81 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>ROB MARCUS</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <h1>
+      <a href="index.html" style="text-decoration: none; color: black">
+        ROB MARCUS
+      </a>
+    </h1>
+    
+    <div id="content">    
+        <div id="container">
+            <div class = "containerBox"> 
+              <div class = "text-box">
+                <container-box-caption>
```

Comment:
I've never actually heard of this html tag and can't seem to find it on Google, could you link me to where you learned about it?  At any rate it doesn't seem to be doing anything different from a \<div\>, so you should prefer to just use \<div\>s since they are well-known, reliable, and consistent across browsers.

Reply:
Think I made it up, oops! Fixed as you suggested.